x/emissions/keeper/keeper.go

```diff
@@ -1595,7 +1594,7 @@ func (k *Keeper) GetStakeReputerAuthority(ctx context.Context, topicId TopicId,
 // Includes the stake placed by delegators on the reputer in that topic.
 func (k *Keeper) SetStakeReputerAuthority(ctx context.Context, topicId TopicId, reputer ActorId, amount cosmosMath.Int) error {
 	key := collections.Join(topicId, reputer)
-	if amount.IsZero() {
+	if amount.IsNil() || amount.IsZero() {
```

Comment:
does nil actually have the same meaning as 0? and not more closer to how we're dealing with NaNs?

Reply:
This is cosmosMath, not alloraMath. In any case maybe it's more of a question for @relyt29 : what's the usage and validation policy regarding `cosmosMath.Int` ?
And then it should be applied to all cases, not just this one; and be included in the Data Validation PR.